### ISSUE TITLE
[MU4 Issue] Accessibility: palette items in More section are not read

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#16092]
**Describe the bug**
It is possible to navigate to and within the More area of a palette, but NVDA and Narrator least do not seem to read these.  Orca does.

**To Reproduce**
1. Start Narrator
2. From any score, hit Shift+Tab to navigate to palettes
3. Navigate to Articulations palette
4. Use expand button to open it
5. Navigate to the More button
6. Hit Space to press it
7. Navigate through this area
9. Hear error: NVDA does not read

**Expected behavior**
These controls should be read.

**Platform information**
 - OS: Windows

**Additional context**
Seems this came up before but I don't see it.  Maybe I'm remembering something else - the discussion of whether the More items should appear in search results? See https://github.com/musescore/MuseScore/issues/11113.

**Comment by jmwmusic** [comment]
This seems to be the same problem as:
#16744 Add Palette not read by screen reader
#25404 Instrument Settings
#16739 Appearance window
I am told all navigate with keyboard, but blind users don't realise this as there is no speech feedback (possibly just with windows) and so are reporting no ability to navigate.  A few more advanced users have got these working with the use of NVDA OCR function.  Not sure if that gives any clues as to why speech is not working.  Most just think it is inaccessible and the only thing that works is Esc to get out.

**Comment by jmwmusic** [comment]
This is now working in 4.7 beta, just 1 little glitch.  When you first open it it seems like you can't move with arrows and nothing reads, but if you shift tab, or tab forward on to the score and back again all works after that.

**Comment by jmwmusic** [comment]
This works correctly with NVDA in 4.7
